<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="colorspace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Color Space</string>
    </property>
    <widget class="QRadioButton" name="binary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="gray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rgb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="print">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="comment_yn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>자막 여부</string>
    </property>
    <widget class="QRadioButton" name="yes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="no">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="content_label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>자막 내용 (English only)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 저장 위치 (English only) (C:/ 으로 시작)</string>
    </property>
   </widget>
   <widget class="QSlider" name="red_bar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>44</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>37</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RED :</string>
    </property>
   </widget>
   <widget class="QSlider" name="green_bar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>96</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN :</string>
    </property>
   </widget>
   <widget class="QSlider" name="blue_bar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>148</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>143</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BLUE :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>4</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>자막 내용 색상</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>213</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_position">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>294</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="red_lcdNumber">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>41</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="green_lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>95</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="blue_lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>148</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="picture"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontext">
   <property name="text">
    <string>text</string>
   </property>
  </action>
  <action name="picture">
   <property name="text">
    <string>picture</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
